docs/onboarding/05-time-series-insights.md

```diff
@@ -0,0 +1,25 @@
+# Chapter 5: Time Series Insights and Azure Digital Twin
+
+- [Overview](https://docs.microsoft.com/en-us/azure/time-series-insights/overview-what-is-tsi) - ~5 min not including video intro, covers basics of IoT data and TSI
+- [Getting Started Tutorial](https://docs.microsoft.com/en-us/azure/time-series-insights/tutorial-set-up-environment)
+- [Storage w/ TSI](https://docs.microsoft.com/en-us/azure/time-series-insights/concepts-storage )
```

Comment:
Should we include the [digital twins how to that covers integrating TSI](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-integrate-time-series-insights)? If so, do you think that best be referenced here or in the learning exercises?